src/index.css

```
@@ -298,10 +299,11 @@
   .dark .glass-panel {
     background: linear-gradient(
       to bottom,
-      rgba(38, 42, 50, 0.60),
-      rgba(28, 32, 38, 0.52)
+      rgba(43, 48, 58, 0.72),
+      rgba(36, 40, 49, 0.68)
     );
-    @apply border border-white/10;
+    @apply border-none;
+    box-shadow: var(--panel-shadow), var(--glass-rim-medium);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)